### PR TITLE
Polyline/polygon parsing adjusted to specification

### PR DESCRIPTION
**Pull request by DmitryMilk**
Dealing with #47 
Adjusted to https://www.w3.org/TR/SVG11/shapes.html#PointsBNF

### CODE REVIEW

File: svgpathtools/svg2paths.py [hunk 1]
```diff
@@ -37,19 +43,21 @@ def ellipse2pathd(ellipse):
...
+    # This check ensures that an n-point polygon is converted to an n-Line path.
+    if is_polygon and closed:
+        points.append(points[0])
 
-    d = 'M' + points.pop(0).replace(',', ' ')
-    for p in points:
-        d += 'L' + p.replace(',', ' ')
-    if closed:
+    d = 'M' + 'L'.join('{0} {1}'.format(x,y) for x,y in points)
+    if is_polygon or closed:
```
Comment (mathandy) [inline svgpathtools/svg2paths.py]: The `or closed` seems a bit dangerous -- the "z" being redundant in this case.  Any argument for why to leave this?

**Comment by mathandy** [comment]
Thanks @DmitryMilk ! The code looks good.  I made one in-line comment/review.

File: svgpathtools/svg2paths.py (hunk 1 above)
Comment (DmitryMilk) [reply svgpathtools/svg2paths.py]: I did not change the logic of parent commit:
- in polygon2paths 'z' was appended in any case (along with appending of an additional segment),
- in polyline2paths 'z' was appended if the ends of the path were equal.
This condition remains that logic.

Comment (mathandy) [reply svgpathtools/svg2paths.py]: OK, perfect! Thanks for the explanation :)

**Pull request merged**